src/style-spec/reference/v8.json

```diff
@@ -6697,14 +6696,6 @@
       "type": "property-type",
       "doc": "Property is interpolable and can be represented using a property expression."
     },
-    "cross-faded": {
```

Comment:
@tristen would you be able to give us insights on how would Studio be impacted by this change?

Reply:
I don't think there should be any impact from a spec perspective. Curious though: how does this change impact things visually on the map?